MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
+    
+    @State var selectedFarmer: FarmerPlace?
+    
+    @Binding var show: Bool
+    
+    var body: some View {
+        
+        Text(selectedFarmer?.name ?? "")
+            .padding()
+        Button {
+            show.toggle()
+            selectedFarmer = nil
+        } label: {
+            Image(systemName: "xmark.circle.fill")
+                .font(.title)
+                .foregroundStyle(.gray, Color(.systemGray6))
```

Comment:
I don't get it haha. Why do you put two colors?
```
.foregroudStyle(.gray)
```
is enought right?

Reply:
It was a test